mysql/infoschema.go

```diff
@@ -375,6 +380,42 @@ func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (forei
 	return foreignKeys, nil
 }
 
+// getIndexes return list all the indexes.
+func getIndexes(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]schema.Index, error) {
+	q := `SELECT DISTINCT INDEX_NAME,COLUMN_NAME,SEQ_IN_INDEX,COLLATION,NON_UNIQUE
+		FROM INFORMATION_SCHEMA.STATISTICS 
+		WHERE TABLE_SCHEMA = ?
+			AND TABLE_NAME = ?
+			AND INDEX_NAME != 'PRIMARY' 
+		ORDER BY INDEX_NAME, SEQ_IN_INDEX;`
+	rows, err := db.Query(q, table.schema, table.name)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var name, column, sequence, collation, nonUnique string
+	indexes := make(map[string]schema.Index)
+	var indexNames []string
+	var indexArr []schema.Index
+	for rows.Next() {
+		if err := rows.Scan(&name, &column, &sequence, &collation, &nonUnique); err != nil {
+			conv.Unexpected(fmt.Sprintf("Can't scan: %v", err))
+			continue
+		}
+		if _, found := indexes[name]; !found {
+			indexNames = append(indexNames, name)
+			indexes[name] = schema.Index{Name: name, Unique: (nonUnique == "0")}
+		}
+		index := indexes[name]
+		index.Keys = append(index.Keys, schema.Key{Column: column, Desc: (collation == "D")})
+		indexes[name] = index
+	}
+	sort.Strings(indexNames)
```

Comment:
Why do we need to sort this, since the query already returns rows sorted by INDEX_NAME?

Reply:
removed sorting, sorting was required for previous implementation, missed to remove the line.